crates/neon-runtime/build.rs

```diff
@@ -26,6 +26,11 @@ fn main() {
 
     // 4. Copy native build artifacts
     copy_build_artifacts(&native_to, &out_dir);
+
+    // Rebuild if any of these environment variables change
```

Comment:
Can you help me understand why this was needed for this PR?

Reply:
It's an existing bug that this PR made apparent. These environment variables impact the compatibility of `libneon`. For example, with these changes `test/dynamic` compiles `neon-runtime` and then `test/electron` tries to use it and fails.